src/test/java/com/google/edith/UserStatsServletTest.java

```diff
@@ -12,7 +12,7 @@
 import com.google.appengine.api.datastore.Query.FilterOperator;
 import com.google.appengine.api.datastore.Query.FilterPredicate;
 import com.google.common.collect.ImmutableMap;
-import com.google.edith.servlets.UserInsights;
+import com.google.edith.servlets.UserInsightsService;
```

Comment:
```UserInsightsService``` does not seem to be used.

If we decided on moving all external dependencies like datasotre to this service then we can mock it while testing the servlet.

Reply:
I removed.